flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/PreAggregationITCase.java

```diff
@@ -711,6 +710,10 @@ public void testMergeInMemory() {
                                     (long) 10101000,
                                     (float) 0,
                                     1.11));
+
+            // projection
+            assertThat(batchSql("SELECT f,e FROM T1"))
```

Comment:
Does this cover the array and list projection?

Reply:
Ditto.